src/App.css

```diff
@@ -119,6 +123,7 @@ input[type="submit"] {
 footer {
   background-color: #47b8d8;
   padding: 20px;
```

Comment:
Personal preference: I prefer using `rem` or `em` for small padding/margin so that their spacings is relative to the text size near them.